llvmlite/binding/newpassmanagers.py

```diff
@@ -27,39 +28,341 @@ class RefPruneSubpasses(IntFlag):
     ALL = PER_BB | DIAMOND | FANOUT | FANOUT_RAISE
 
 
-class ModulePassManager(ffi.ObjectRef):
+def get_module_level_passes():
```

Comment:
Is this useful for anything? (Similar question for the function-level passes method below)